extension/app/ts/AddressBook.tsx

```diff
@@ -382,77 +363,74 @@ export function AddressBook() {
 
 	return (
 		<main>
```

Comment:
```suggestion
		<main class = 'max-width: 1920px; margin: auto;' >
```
May be a good idea to limit the width of the content area so content so lengths of wide elements don't start looking bad on wider displays.

Could set the styles in the extension pages ``/extension/app/html/addressBook.html`` instead of the component instead I just don't know how to suggest there.

![image](https://user-images.githubusercontent.com/52896585/213940577-cd8a68bd-de83-4f26-a0ca-d219b6f49d8d.png)
![image](https://user-images.githubusercontent.com/52896585/213940640-99fca483-28fc-49fc-ac9b-0a9217daf4b4.png)

Reply:
> May be a good idea to limit the width of the content area so content so lengths of wide elements don't start looking bad on wider displays.

I pretty strongly disagree with this.  The user should be the one ultimately in control of how wide of a window they want, as they can shrink their window if they desire.  By hard coding a limit, you are constraining sizing options available to the user and helping the subset of users who have particularly wide monitors and do not know how to resize a window.  I am of the belief that this set of users is incredibly small and should not be optimized for, especially at the cost of flexibility/responsiveness to other users who *want* to have a very wide view of things.

Reply:
I guess we could limit the amount on how much the account entry grows to the size where there is no benefit for it to grow anymore (everything fits already

Reply:
Yeah I would be OK with "fit to content" and the content boxes are scaled to fit their content etc.  I am just against a fixed pixel or % width is all.